lib/assets/gallery-video-article.mst

```diff
@@ -62,6 +62,8 @@ let article_content = document.getElementById('article-content');
 if (article_content != null) {
     if (article_content.children.length < 1) {
         document.getElementById('article-title').className += ' less-margin-bottom';
+        document.getElementById('article-support').className += ' width-full';
```

Comment:
Why not use toggle?
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList -> toggle() method